va/va_dec_hevc.h

```diff
@@ -327,7 +484,77 @@ typedef struct  _VASliceParameterBufferHEVC
     uint32_t                va_reserved[VA_PADDING_LOW];
 } VASliceParameterBufferHEVC;
 
+/**
+ * \brief HEVC Extented Slice Parameter Buffer Structure For Long Format
+ *
+ * This data structure contains extension profiles (range extension and screen content).
+ *
+ * VASliceParameterBufferHEVCRext structure should be accompanied by a
+ * slice data buffer, which holds the whole raw slice NAL unit bit streams
+ * including start code prefix and emulation prevention bytes not removed.
+ *
+ * This structure conveys parameters related to slice segment header and should
+ * be sent once per slice with VASliceParameterBufferHEVC in a single buffer.
+ *
+ * For short format, this data structure is not sent by application.
+ *
+ */
+typedef struct  _VASliceParameterBufferHEVCRext
+{
+    /* below four parameters are used to replace data types of the
+     * corresponding parameters of those in \# VASliceParameterBufferHEVC.
+     */
+    int16_t                 luma_offset_l0[15];
+    int16_t                 ChromaOffsetL0[15][2];
+    int16_t                 luma_offset_l1[15];
+    int16_t                 ChromaOffsetL1[15][2];
+
+    union
+    {
+        uint32_t            value;
+        struct
+        {
+            uint32_t        cu_chroma_qp_offset_enabled_flag    : 1;
+            uint32_t        use_integer_mv_flag                 : 1;
+            /** \brief Reserved bytes for future use, must be zero */
+            uint32_t        reserved                            : 30;
+        } fields;
```

Comment:
To keep the same code style, could you please change the name from fields to bits and move 'uint32_t value;' after this structure.